enhancements/cluster-logging/cluster-logging-es-rollover-data-design.md

```diff
@@ -99,9 +99,9 @@ metadata:
 spec:
   logStore:
     retentionPolicy: 
-      logs.app:
+      logs-app:
```

Comment:
This can remain "dots" as this is the api... and not what is translated to actual alias.  Though @alanconway can you review maybe and suggest changes we should make for 4.5 that jive with ClusterLogForwarder

Reply:
I changed the names from log.app to "Infrastructre", "Application" and "Audit" based on https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md
In particular those conventions use CapsMixedCase for enum values and recommend avoiding abbreviations.